kubetest/client.py

```diff
@@ -215,6 +216,48 @@ def load_service(self, path, set_namespace=True):
             service.namespace = self.namespace
         return service
 
+    def load_PersistentVolume(self, path, set_namespace=False):
```

Comment:
I think we want to avoid the camel case here, most other `load_` methods like `local_rolebinding` use full lower case.